application/ImageButton.cpp

```diff
@@ -22,7 +22,7 @@
 using namespace std;
 
 ImageButton::ImageButton(BString imageName,BMessage *message,
-                        float marginProportion, int frameBehaviour,
+                        float marginProportion, int frameBehaviour, const char* tooltip,
                         const char* name,  uint32 flags)
```

Comment:
You can move the new parameter to the front of the next line to avoid line overflow.